lisa/tools/ls.py

```diff
@@ -50,6 +50,17 @@ def list_dir(self, path: str, sudo: bool = False) -> List[str]:
         else:
             return []
 
+    def list_files(self, path: str, sudo: bool = False) -> List[str]:
```

Comment:
How about use `ls` cmd to list file only? It doesn't need to run two commands to query files.

Reply:
I couldn't find a straightforward way to do this. Let me see if I can find any behaviour specific to folders output